### PR TITLE
Redesign activities

### PR DESCRIPTION
**Pull request by Yelinz**

### CODE REVIEW

**Comment by czosel** [comment]
@Yelinz can you resolve the conflicts?

**Comment by Yelinz** [comment]
Resolved conflicts

File: app/index/activities/controller.js [hunk 1]
```diff
@@ -45,5 +46,17 @@ export default Controller.extend({
...
+  sortedActivities: sort('activities', function(a, b) {
+    let dateA = a.get('from').toDate()
+    let dateB = b.get('from').toDate()
+    if (dateA > dateB) {
+      return -1
+    }
+    if (dateA < dateB) {
+      return 1
+    }
+    return 0
```
[the remark this answers is not in the document]
Comment (Yelinz) [reply app/index/activities/controller.js]: Changed it to the one liner.

File: app/index/activities/edit/controller.js [hunk 2]
```diff
@@ -15,49 +14,18 @@ import moment from 'moment'
...
-      }, moment.duration())
-  }),
-
   /**
    * Whether the save button is enabled
    *
-   * This is true if the activity and all its block is valid and there are some
-   * kind of changes on the activity or its blocks
+   * This is true if the activity is valid and there are some
+   * kind of changes on the activity
```
Comment (czosel) [inline app/index/activities/edit/controller.js]:
```suggestion
   * changes on the activity
```

File: app/index/activities/route.js [hunk 3]
```diff
@@ -33,13 +33,17 @@ export default Route.extend(RouteAutostartTourMixin, {
...
+
   /**
    * Setup controller hook, set the current user
    *
    * @method setupContrller
    * @param {Ember.Controller} controller The controller
-   * @public
-   */
+     * @public
+     */
```
Comment (czosel) [inline app/index/activities/route.js]: Broken indentation

File: app/index/activities/route.js.orig [hunk 4]
```diff
@@ -0,0 +1,265 @@
+/**
```
Comment (czosel) [inline app/index/activities/route.js.orig]: I think this file should be removed

File: mirage/factories/activity.js.orig [hunk 5]
```diff
@@ -0,0 +1,56 @@
+import { Factory, faker, trait } from 'ember-cli-mirage'
```
Comment (czosel) [inline mirage/factories/activity.js.orig]: same here

File: app/index/activities/route.js.orig (hunk 4 above)
Comment (Yelinz) [reply app/index/activities/route.js.orig]: All ```.orig``` files already removed

File: app/index/activities/route.js (hunk 3 above)
Comment (Yelinz) [reply app/index/activities/route.js]: Fixed

File: tests/test-helper.js [hunk 6]
```diff
@@ -3,7 +3,7 @@ import { setResolver } from 'ember-mocha'
 import { mocha } from 'mocha'
 
 mocha.setup({
-  timeout: 10000,
+  timeout: 100000,
```
[the remark this answers is not in the document]
Comment (Yelinz) [reply tests/test-helper.js]: Reverted

**Pull request merged**